region/norfair/east.json

```diff
@@ -222,6 +243,21 @@
                       ]},
                     {"heatFrames": 300}
                   ]
+                },
+                {
+                  "name": "PreCathedral Left Door Frozen Sova Step",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateHeatRooms", 
+                    "canUseFrozenEnemies",
+                    "canTrickyJump",
+                    {"heatFrames": 900}
+                  ],
+                  "note": [
+                    "Guide the morph tunnel Sova through the shot blocks by keeping it on camera.",
+                    "The shot blocks must be cleared so it may continue down the right wall.",
+                    "Freeze the enemy as it touches the 2nd downward slope looking wall piece."
```

Comment:
It's a bit tricky. It might be worth mentioning if it's failed then the global Sova could be used on the left wall as a backup?